client/src/panels/panel_declarations/covid/covid_estimates.js

```diff
@@ -512,6 +340,209 @@ class CovidEstimatesPanel extends React.Component {
+      <WrappedNivoBar
+        data={graph_data_arr}
+        keys={graph_keys}
+        indexBy={graph_index_key}
+        colors={(d) => colors(d.id)}
+        margin={{
+          top: 50,
+          right: 40,
+          bottom: 120,
+          left: 40,
+        }}
+        bttm_axis={{
+          format: (d) =>
+            _.words(d).length > 3 ? d.substring(0, 20) + "..." : d,
+          tickSize: 3,
+          tickRotation: -45,
+          tickPadding: 10,
+        }}
+        graph_height="450px"
+        enableGridX={false}
+        remove_left_axis={true}
+        theme={{
+          axis: {
+            ticks: {
+              text: {
+                fontSize: 12,
+                fill: textColor,
+                fontWeight: "550",
+              },
+            },
+          },
+        }}
+      />
+    );
+
+    const additional_text_args = (() => {
+      if (subject.level === "gov") {
+        return {
+          covid_est_pct_of_all_est:
+            panel_args[`gov_covid_estimates_in_year`] /
+            panel_args[`gov_total_estimates_in_year`],
+        };
+      } else {
+        const dept_covid_estimates_in_year = _.reduce(
+          sorted_data,
+          (memo, { stat, vote }) => memo + vote + stat,
+          0
+        );
+
+        return {
+          dept_covid_estimates_in_year,
+        };
+      }
+    })();
+
+    return (
+      <div className="row align-items-center">
+        <div className="col-12 col-lg-6 medium-panel-text">
+          <TM
+            k={`covid_estimates_summary_text_${subject.level}`}
+            args={{ ...panel_args, ...additional_text_args }}
+          />
+          <TM k={"covid_estimates_by_release_title"} />
+          <ul>
+            {_.map(sorted_data, ({ est_doc, vote, stat }) => (
+              <li key={est_doc}>
+                <TM
+                  k={"covid_estimates_by_release"}
+                  args={{
+                    est_doc_name: get_est_doc_name(est_doc),
+                    est_doc_glossary_key: get_est_doc_glossary_key(est_doc),
+                    total: vote + stat,
+                  }}
+                  el="span"
+                  style={{ display: "inline-block" }}
+                />
+                {get_tooltip(
+                  "est_doc_total",
+                  selected_year,
+                  subject.id,
+                  est_doc
+                )}
+              </li>
+            ))}
+          </ul>
+        </div>
+        <div className="col-12 col-lg-6">
+          {!is_a11y_mode && (
+            <StandardLegend
+              items={legend_items}
+              isHorizontal={true}
+              onClick={(label) => {
+                !(
+                  this.state.selected.length === 1 &&
+                  this.state.selected.includes(label)
+                ) &&
+                  this.setState({
+                    selected: toggle_list(this.state.selected, label),
+                  });
```

Comment:
Just checking my understanding, `SummaryTab` and `tab_content_configs` were moved to being properties of `CovidEstimatesPanel` so that `SummaryTab` could call `setState` on `CovidEstimatesPanel`, right? Were there other reasons?

Reply:
Yeah that’s all it was